src/DiagResponse.php

```diff
@@ -2,22 +2,15 @@
 
 namespace Diag;
 
-class DiagResponse
-{
-    private $statusCode;
-    private $content;
+use Symfony\Component\HttpFoundation\JsonResponse;
 
-    public function __construct($content, $statusCode = 200)
-    {
-        $this->content = $content;
-        $this->statusCode = $statusCode;
-    }
-
-    public function __toString()
+class DiagResponse extends JsonResponse
+{
+    public function __construct($data = null, int $status = 200, array $headers = array(), bool $json = false)
```

Comment:
Copied from php 5.3? The array syntax gives it away.

Reply:
nah. already cleaned;